### PR TITLE
Docs authoring pack

### PR DESCRIPTION
**Pull request by IEvangelist**
Hey @Jim-Parker,

Do you know why this build failed?

### CODE REVIEW

**Comment by Jim-Parker** [comment]
@IEvangelist i reran it. it's okay now. 
Can you clarify what you're doing with this PR?

**Comment by IEvangelist** [comment]
> @IEvangelist i reran it. it's okay now.
> Can you clarify what you're doing with this PR?

Sorry, yes... I'm doing what you asked of me and more. You said,  "@IEvangelist would you be willing to make this same update the external contributor guide version of the article? It would make sense to keep these in sync." I'm updating both docs and actually syncing up the Docs Authoring Pack bits as well, as those are public facing.

Related to https://github.com/MicrosoftDocs/docs-help-pr/pull/2912

File: Contribute/code-in-docs.md [hunk 1]
```diff
@@ -197,13 +197,16 @@ If you use a language or environment word after the triple-backticks (\`\`\`) th
 that word appears in the code section title bar on the rendered page. Whenever possible, use a language or
 environment indicator in your inline code blocks.
 
+> [!NOTE]
```
Comment (tdykstra) [inline Contribute/code-in-docs.md]: The curly quote examples got changed to normal quotes.

**Comment by Jim-Parker** [comment]
@IEvangelist thanks for the clarification. the reason I'm asking - since keeping the external guide in sync with our internal guidance is entirely manual (for now) we're taking a minimal approach to the external guide. I should have mentioned this when I made the initial request. 
Does that perspective change what you think should be added?

**Comment by IEvangelist** [comment]
> @IEvangelist thanks for the clarification. the reason I'm asking - since keeping the external guide in sync with our internal guidance is entirely manual (for now) we're taking a minimal approach to the external guide. I should have mentioned this when I made the initial request.
> Does that perspective change what you think should be added?

No, my perspective doesn't change. I was aware of the external guide, as I asked to update it earlier and never had time to do so. All of these updates are valid and should be externally available.

**Comment by Jim-Parker** [comment]
@IEvangelist I merged based on our conversation, but then noticed you might still be making changes. For the external guide, I still need to merge to Live, so let me know if you have more to do here and I'll hold off on that.

**Comment by Jim-Parker** [comment]
@IEvangelist also, there's an invalid link in image-compression.md file.

**Pull request merged**